cmd/admin-bucket-handlers.go

```diff
@@ -213,8 +213,9 @@ func (a adminAPIHandlers) ListRemoteTargetsHandler(w http.ResponseWriter, r *htt
 
 	defer logger.AuditLog(ctx, w, r, mustGetClaimsFromToken(r))
 	vars := mux.Vars(r)
-	bucket := path.Clean(vars["bucket"])
+	bucket := vars["bucket"]
```

Comment:
```suggestion
	bucket := vars["bucket"]
```

We still need to path.Clean this to avoid encrypt errors if any. @poornas